### DISCUSSION TITLE
Functionality for PMDG, H145?

### DISCUSSION THREAD

**Discussion by BlipMyRadar** [calibx/msfsdeck#227] [Ideas]
Hey, firstly I just want to say **WOW**, I am very impressed so far! Thanks a bunch for creating this but most importantly, updating and troubleshooting with us all.

Admittedly I have only owned the Loupedeck for less than a day so perhaps I am not yet familiar with its functions, however, I cant get it to work for autopilot functionality in the PMDG 737. 

Similarly in the Hype Performance Groups' H-145 helicopter. I wonder if I am not using it correctly? For both these two I have tried, some lights don't seem to work either.

Works perfectly for the F-35 by India Fox, I am yet to try others.

I have seen that you have mentioned in other Q&A's that it is how the controls are implemented within the sim, and dependant on aircraft. If that is so, do you think there could be a workaround? I'm assuming it would require delving into config files and such. 

Again, **thanks a bunch**. I look forward to hearing back from you. 🥇

**Comment by smoothchat** [comment]
I think the plugin uses the established Microsoft/Asobo commands whereas PMDG use non standard commands for most things in the aircraft.   

You might like to consider investing in SPAD.next.  Among many things, It can take input from the loupedeck and you can configure it to convert it to the codes recognised by PMDG. There is a myriad of other things SPAD.next will allow you to do. It is very powerful so there will be a necessary learning curve.

**Comment by calibx** [comment]
In 3.0 I have included a misc input to allow more configurable inputs but to allow all oh them I need to include a WASM module (as spadnext, FSUIPc or Mobiflight). I will work again on that when the 4.0 (loupedeck 6 ans and msfs 2024 compatibility) will be released

**Comment by smoothchat** [comment]
Thanks.  Some other things like some of the light buttons and the weather also need to be looked at for 2024.  Thanks for making this wonderful addon.   Cheers.
